relay-general/src/protocol/stacktrace.rs

```diff
@@ -93,17 +95,19 @@ pub struct Frame {
     pub pre_context: Annotated<Array<String>>,
 
     /// Source code of the current line (`lineno`).
+    #[metastructure(skip_serialization = "null")]
     pub context_line: Annotated<String>,
 
     /// Source code of the lines after `lineno`.
-    #[metastructure(skip_serialization = "empty")]
+    #[metastructure(skip_serialization = "null")]
```

Comment:
`empty` is fine for the arrays, especially since you only changed it for `post_context` but not `pre_context`. Let's stick with `empty`.